constructs/Compute/virtualMachinesMultiple/main.bicep

```diff
@@ -230,11 +230,6 @@ param extensionCustomScriptProtectedSetting object = {}
 @description('Optional. Location for all resources.')
```

Comment:
There are currently a truckload of conflicts due to the file genreations. I'll re-generate all once the review is done.
cc: @ahmadabdalla

Reply:
A couple of readmes need to be regenerated:

![image](https://github.com/Azure/ResourceModules/assets/28486158/58a76a5c-0341-4384-889a-8b9a4cdd0ba0)

Reply:
removed